common_test.go

```diff
@@ -171,10 +212,12 @@ func createKeyspace(tb testing.TB, cluster *ClusterConfig, keyspace string, disa
 		'replication_factor' : %d
 	}`, keyspace, *flagRF)
 
-	if disableTablets {
-		query += " AND tablets = {'enabled': false}"
-	} else {
-		query += " AND tablets = {'enabled': true, 'initial': 8};"
+	if checkIfClusterSupportsTablets() {
+		if disableTablets {
+			query += " AND tablets = {'enabled': false}"
+		} else {
+			query += " AND tablets = {'enabled': true, 'initial': 8};"
```

Comment:
I don't think we should set the 'initial', just use the defaults.

Reply:
(I know it's not related to your change, but worth fixing it)

Reply:
done